Source/WebCore/editing/Editing.cpp

```diff
@@ -574,6 +574,11 @@ Element* enclosingAnchorElement(const Position& p)
     return nullptr;
 }
 
+Element* enclosingAtomicNode(const Position&)
+{
+    return enclosingNodeOfType(m_selectionToDelete.start(), isAtomicNode);
```

Comment:
m_selecionToDelete isn't defined here. You need to replace it with whatever name you give it to the argument.